packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -33,6 +32,7 @@ contract FederatedAttestations is
   using SafeMath for uint256;
```

Comment:
After Solidity 0.8.0 SafeMath is no longer needed as overflow checking is built in by default
https://ethereum.stackexchange.com/questions/91367/is-the-safemath-library-obsolete-in-solidity-0-8-0

Reply:
I think we are stuck on version 0.5.13 in order to use the current monorepo release process & if we want this to be part of the core contracts 🤔  do you know a way around this?